### ISSUE TITLE
Couple of questions regarding proper use of this plugin

### ISSUE THREAD

**Issue by hapklaar** [jjgraphix/PressureAdvanceSettingPlugin#1]
Should the pressure_advance value be removed from Klipper's printer.cfg to use this, or does the value specified in Cura alway override this?

What happens if I leave the value at 0 in Cura, will Klipper still use the value in printer.cfg or will it be forced to '0'?

Thanks for creating this!

**Comment by kravviec** [comment]
can't find any pressure advance command in g-code from Cura 4.13 so I created M900 macro, with K-factor from Marlin's  linear advance plugin

**Comment by jjgraphix** [comment]
Sorry for the long delay! I finally published a new version that includes a lot of improvements. In the original version there was no way to disable the command entirely without turning off the plugin but this is now resolved. 

Now if you disable **Pressure Advance Control** no new commands are inserted into gcode. If you wish to turn off pressure advance, _enable_ control then set values to "0". This will instruct Klipper to disable it. Your config value never changes and is simply the default used whenever Klipper is restarted.

Let me know if you have questions about the new version.
